sound/soc/sof/imx/imx8m.c

```diff
@@ -288,6 +288,96 @@ static struct snd_soc_dai_driver imx8m_dai[] = {
 },
 };
 
+int imx8m_dsp_set_power_state(struct snd_sof_dev *sdev,
```

Comment:
comment on commit message

"
when someone opens the ALSA sound card (aplay/arecord, etc) SOF core
   calls PM runtime resume to turn on the DSP
 - when the ALSA sound card is closed SOF core make use of PM subsystem
  to call PM runtime suspend and thus turning off the DSP.
"

SOF core -> ASoC core. the SOF core only registers a component, the pm_runtime_get_sync() is done at a higher level.